core/iam_authenticator_test.go

```diff
@@ -442,6 +504,10 @@ func TestIamGetTokenSuccessRT(t *testing.T) {
 				"expiration": %d,
 				"refresh_token": "%s"
 			}`, iamAuthTestAccessToken2, expiration, newRefreshToken)
+			username, password, ok := r.BasicAuth()
```

Comment:
since this test is actually relying on the refresh token functionality, I just added a check to the mock server-side code to make sure we're passing in bx:bx in the basic auth header